addons/im_livechat/views/im_livechat_channel_views.xml

```diff
@@ -23,48 +23,37 @@
                     <field name="rating_count"/>
-                    <field name="image_128"/>
                     <templates>
                         <t t-name="kanban-menu">
                             <div class="container">
                                 <a type="object" name="action_view_rating" class="dropdown-item" role="menuitem">Ratings</a>
                                 <a type="open" class="dropdown-item" role="menuitem">Configure Channel</a>
                             </div>
                         </t>
-                        <t t-name="kanban-box">
-                            <div class="oe_kanban_global_click px-4" t-att-class="{'o-livechat-ChannelKanban-highlighted': record.available_operator_ids.raw_value.length > 0}">
-                                <div class="o_kanban_image" t-if="record.image_128.raw_value">
-                                    <img t-att-src="kanban_image('im_livechat.channel', 'image_128', record.id.raw_value)" class="img-fluid" alt="Channel"/>
-                                </div>
-                                <div class="oe_kanban_details">
-                                    <div class="d-flex justify-content-between">
-                                        <div>
-                                            <field name="name" class="fs-4" style="word-wrap: break-word;"/>
-                                            <p class="fst-italic fs-5"><t t-esc="record.nbr_channel.raw_value"/> Sessions</p>
-                                        </div>
-                                        <div>
-                                            <button t-if="record.are_you_inside.raw_value" name="action_quit" type="object" class="btn btn-primary">Leave</button>
-                                            <button t-if="!record.are_you_inside.raw_value" name="action_join" type="object" class="btn btn-secondary">Join</button>
-                                        </div>
+                        <t t-name="kanban-card" class="p-0 row g-0">
+                            <aside t-if="record.image_128.raw_value" class="ps-4 col-3" t-att-class="{'o-livechat-ChannelKanban-highlighted': record.available_operator_ids.raw_value.length > 0}">
+                                <field  name="image_128" widget="image" options="{'img_class': 'object-fit-contain'}" alt="Channel"/>
+                            </aside>
+                            <main class="pe-4 py-2 ps-2 col" t-att-class="{'o-livechat-ChannelKanban-highlighted': record.available_operator_ids.raw_value.length > 0, 'ps-4': !record.image_128.raw_value}">
+                                <div class="d-flex justify-content-between">
+                                    <div>
+                                        <field name="name" class="fs-4" style="word-wrap: break-word;"/>
+                                        <p class="fst-italic fs-5"><field name="nbr_channel"/> Sessions</p>
```

Comment:
Already there before diff, but `<p><t-esc=""/> Sessions</p>` is not translation-friendly. Translators see ` Sessions`, they do not know there's a number before, and also some languages do not necessarily would like to start with digit. Not an Irish expert, but based on this wiki Irish needs to start with "a" to be gramatically correct, e.g. `a 10 seisiún`.

Not a huge deal though. We can keep like this for now.